packages/edit-site/src/components/editor/index.js

```diff
@@ -200,6 +201,21 @@ function Editor() {
 											<SidebarComplementaryAreaFills />
 											<InterfaceSkeleton
 												labels={ interfaceLabels }
+												drawer={
```

Comment:
I think we might want to add a custom label for the drawer in `interfaceLabels`.

Like, the interface element is called "Drawer" but it's filled by "Navigation Sidebar" (or at least I think that's the point of `interfaceLabels` 😄).

Reply:
Good catch!